src/sentry/static/sentry/app/views/organizationGroupDetails/groupMerged/mergedItem.tsx

```diff
@@ -78,27 +79,28 @@ const MergedItem = createReactClass({
 
     // clicking anywhere in the row will toggle the checkbox
     GroupingActions.toggleUnmerge([fingerprint, event.id]);
-  },
+  };
 
   handleCheckClick() {
```

Comment:
What about this one too?

Reply:
this function is empty... I would like to remove it, but in the end, I decided to change only the necessary for the conversion :)